axe.d.ts

```diff
@@ -408,6 +408,24 @@ declare namespace axe {
     boundingClientRect: DOMRect;
   }
 
+  type GridCell = VirtualNode[];
+
+  interface Grid {
+    container: VirtualNode | null;
+    cells: GridCell[] & { _negativeIndex?: number };
```

Comment:
`cells` is a list of rows, not a list of cells, no?

```suggestion
  type GridCell = VirtualNode[];
  type GridCellRow = GridCell[] & { _negativeIndex?: number };
  type GridCellMatrix = GridCellRow[] & { _negativeIndex?: number };

  interface Grid {
    container: VirtualNode | null;
    cells: GridCellMatrix;
```

Reply:
Or if you prefer,

```suggestion
  type SparseMatrixRow<T> = T[] & { _negativeIndex?: number };
  type SparseMatrix<T> = SparseMatrixRow<T>[] & { _negativeIndex?: number };

  type GridCell = VirtualNode[];
  interface Grid {
    container: VirtualNode | null;
    cells: SparseMatrix<GridCell>;
```

Reply:
I'm just going to ditch the private pro. Shouldn't be using that anyway. The whole reason to add `getNodeGrid` is to avoid private props.

Reply:
Unknown's fine